tasmoadmin.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tasmoadmin",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/tprelog/iocage-tasmoadmin.git", 
+  "official": "false",
+  "properties": {
+    "allow_raw_sockets": "1",
+    "dhcp": "1"
+  },
+  "pkgs": [
+    "shells/bash",
```

Comment:
Hi,

categories can be removed iocage detects that automatically now, the pkg-list.json is also no needed. Could u please fix that for all ur submissions. Thanks.

Reply:
Fixed for all submissions!

Reply:
thanks!